### ISSUE TITLE
[wct] Error is thrown on spread operator

### ISSUE THREAD

**Issue by osamamaruf** [Polymer/tools#791]
While running tests the below error is being faced:
```
Failed to parse file: D:\repo\abc\abc-ui/src/common/components/abc-payment-workflow/abc-payment-workflow.js
Error: Line 127: Unexpected token ...
    at ErrorHandler.constructError (D:\repo\abc\abc-ui\node_modules\esprima\dist\esprima.js:5004:22)
    at ErrorHandler.createError (D:\repo\abc\abc-ui\node_modules\esprima\dist\esprima.js:5020:27)
    at Parser.unexpectedTokenError (D:\repo\abc\abc-ui\node_modules\esprima\dist\esprima.js:1985:39)
    at Parser.throwUnexpectedToken (D:\repo\abc\abc-ui\node_modules\esprima\dist\esprima.js:1995:21)
    at Parser.parseObjectPropertyKey (D:\repo\abc\abc-ui\node_modules\esprima\dist\esprima.js:2492:33)
    at Parser.parseObjectProperty (D:\repo\abc\abc-ui\node_modules\esprima\dist\esprima.js:2527:25)
    at Parser.parseObjectInitializer (D:\repo\abc\abc-ui\node_modules\esprima\dist\esprima.js:2595:35)
    at Parser.inheritCoverGrammar (D:\repo\abc\abc-ui\node_modules\esprima\dist\esprima.js:2278:37)
    at Parser.parsePrimaryExpression (D:\repo\abc\abc-ui\node_modules\esprima\dist\esprima.js:2347:38)
    at Parser.inheritCoverGrammar (D:\repo\abc\abc-ui\node_modules\esprima\dist\esprima.js:2278:37)
Failed to parse file: D:\repo\abc\abc-ui/myapp.js
Error: Line 127: Unexpected token ...
    at ErrorHandler.constructError (D:\repo\abc\abc-ui\node_modules\esprima\dist\esprima.js:5004:22)
    at ErrorHandler.createError (D:\repo\abc\abc-ui\node_modules\esprima\dist\esprima.js:5020:27)
    at Parser.unexpectedTokenError (D:\repo\abc\abc-ui\node_modules\esprima\dist\esprima.js:1985:39)
    at Parser.throwUnexpectedToken (D:\repo\abc\abc-ui\node_modules\esprima\dist\esprima.js:1995:21)
    at Parser.parseObjectPropertyKey (D:\repo\abc\abc-ui\node_modules\esprima\dist\esprima.js:2492:33)
    at Parser.parseObjectProperty (D:\repo\abc\abc-ui\node_modules\esprima\dist\esprima.js:2527:25)
    at Parser.parseObjectInitializer (D:\repo\abc\abc-ui\node_modules\esprima\dist\esprima.js:2595:35)
    at Parser.inheritCoverGrammar (D:\repo\abc\abc-ui\node_modules\esprima\dist\esprima.js:2278:37)
    at Parser.parsePrimaryExpression (D:\repo\abc\abc-ui\node_modules\esprima\dist\esprima.js:2347:38)
    at Parser.inheritCoverGrammar (D:\repo\abc\abc-ui\node_modules\esprima\dist\esprima.js:2278:37)
```

### Versions & Environment

- Polymer CLI:  v1.9.1
- node: v8.11.3 (Windows)
- Operating System: Windows 7
- Browsers Tested on : Chrome v.70
#### Steps to Reproduce

1. Execute unit tests

#### Expected Results
There should not be any error in console.

#### Actual Results
Above error is faced.

**Comment by TimvdLippe** [comment]
Same issue as #720

**Comment by osamamaruf** [comment]
@TimvdLippe in my case the build was successful **but** the test cases throw an error. Is it related to #720 ?

**Comment by TimvdLippe** [comment]
@osamamaruf In that case you are not fully transpiling your source. The browsers do not implement object spread yet, thus they will throw a parse error. To fix that, make sure your transpilation configuration is correct and that ES6/5 code only is generated for the browser.

**Comment by osamamaruf** [comment]
> In that case you are not fully transpiling your source

@TimvdLippe  For build I am using the provided preset for es5 i.e. `"preset": "es5-bundled",` and this code is only generated for the browser and works fine in IE and chrome.

However, the issue I am facing is while running `wct --skip-selenium-install` command. 

>  The browsers do not implement object spread yet, thus they will throw a parse error. 

I have specified the `chrome` as the browser on which the tests will be executed where the chrome version is `70` which **does** have spread operator support. My wct browser configuration is:

```
chrome: [
                    'window-size=1920,1080',
                    'headless',
                    'disable-gpu',
                    'no-sandbox'
                ]

```

The issue I have raised is specifically related to **error** thrown during `unit tests` execution. Is is still related to #720 ? Or should this be reopened?
